### DISCUSSION TITLE
Add twitter stream to the side bar

### DISCUSSION THREAD

**Discussion by anuraj-rp** [mmistakes/minimal-mistakes#3071] [Q&A]
I want to add my twitter stream to the sidebar using this plugin - https://github.com/rob-murray/jekyll-twitter-plugin

I used the starter template mm-github-pages-starter for my website. 

What is the best way to go about this?

**Comment by mmistakes** [comment]
If you're hosting with GitHub Pages there's a good chance that plugin isn't supported as only a handful are allowed for security reasons.
If you can use the plugin, then take a look at the [sidebar](https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/sidebar.html) or [author-profile](https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/author-profile.html) includes.

You'll need to customize one of those. To do that copy from this repo and place in your repo in an `_includes` folder... keep the same filename. Jekyll will use your local versions instead of the ones bundled in the remote theme.

Then make modifications to the sidebar or author include following whatever installation instructions the plugin provides. Looks like you just drop in a Liquid tag where you want the stream to appear.
